src/app/zap-templates/zcl/data-model/chip/operational-state-oven-cluster.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="OperationalStateEnum" type="enum8">
+    <cluster code="0x0048"/>
+    <item name="Running"          value="0x01"/>
```

Comment:
This overlaps with the base cluster.  I know the spec has this, but it shouldn't.  Please make sure a spec issue is filed.

Reply:
Conformance in the cluster is `X` which seems to be `This is disallowed for the cluster derivation.`.

I believe this should be removed from the xml.